src/containers/App.tsx

```diff
@@ -33,81 +30,51 @@ import { VaiContextProvider } from '../context/VaiContext';
-    super(props);
-    this.state = {
-      lang: initialLang,
-    };
-  }
-
-  render() {
-    const { lang } = this.state;
-    const message = messages[lang];
-    return (
-      <Theme>
-        <IntlProvider locale={lang} messages={message}>
-          <Web3Wrapper>
-            <QueryClientProvider client={queryClient}>
-              <Provider store={store}>
-                <RefreshContextProvider>
-                  <VaiContextProvider>
-                    <MarketContextProvider>
-                      <MuiThemeProvider>
-                        <AuthProvider>
-                          <BrowserRouter>
-                            <ToastContainer
-                              autoClose={8000}
-                              transition={Slide}
-                              hideProgressBar
-                              newestOnTop
-                              position={toast.POSITION.TOP_LEFT}
-                            />
-                            <Layout>
-                              <Switch>
-                                <Route exact path="/dashboard" component={Dashboard} />
-                                <Route exact path="/vote" component={Vote} />
-                                <Route exact path="/xvs" component={XVS} />
-                                <Route exact path="/market" component={Market} />
-                                <Route exact path="/transaction" component={Transaction} />
-                                <Route exact path="/vault" component={Vault} />
-                                <Route exact path="/market/:asset" component={MarketDetail} />
-                                <Route
-                                  exact
-                                  path="/vote/leaderboard"
-                                  component={VoterLeaderboard}
-                                />
-                                <Route exact path="/vote/proposal/:id" component={VoteOverview} />
-                                <Route
-                                  exact
-                                  path="/vote/address/:address"
-                                  component={ProposerDetail}
-                                />
-                                <Route exact path="/convert-vrt" component={VrtConversion} />
-                                {isOnTestnet && <Route exact path="/faucet" component={Faucet} />}
-                                <Redirect from="/" to="/dashboard" />
-                              </Switch>
-                            </Layout>
-                          </BrowserRouter>
-                        </AuthProvider>
-                      </MuiThemeProvider>
-                    </MarketContextProvider>
-                  </VaiContextProvider>
-                </RefreshContextProvider>
-              </Provider>
-            </QueryClientProvider>
-          </Web3Wrapper>
-        </IntlProvider>
-      </Theme>
-    );
-  }
-}
+const App: React.FC = () => (
```

Comment:
In this file we can just delete `react-intl` and then in `pages/App.tsx` we can setup the `react-i18next`. This file will be eventually deleted and we'll remove the flag for running in dev and the root of the application will be `src/pages/App.tsx`

Reply:
Ok got you, that makes sense. I've made the update.